### PR TITLE
chore: Update Annual purchase test cases

### PR DESCRIPTION
**Pull request by yasserfaraazkhan**
Dev PR: https://github.com/mattermost/mattermost-webapp/pull/11985

<a href="https://gitpod.io/#https://github.com/mattermost/mattermost-test-management/pull/61"></a>

### CODE REVIEW

File: data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md [hunk 1]
```diff
@@ -42,19 +42,24 @@ steps_hashed: f690569e888bbcc689e9149511b902ca20999833a5f8e71942c37778242c459ad4
 Tests Steps\
 \
 **For yearly option**\
-1\. By default 'Yearly' subscription should be selected\
+1\. By default customer making a purchase for the first time will be subscribing to 'Yearly' subscription\
 \
 2\. Provision to add seats >=current active users. Anything less than 10 should show user error\
 \
 3\. 'Yearly Savings' should be calculated based on No. Of seats and cost per seat. This should be displayed\
```
Comment (marianunez) [inline data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:49]: Yearly savings is no longer available and the screenshots below no longer apply given that we don't have the toggle.

**Review by marianunez — COMMENTED** [review]
@yasserfaraazkhan not sure if you will have this in another PR but we also need to add the new notifications current monthly customers will have https://mattermost.atlassian.net/browse/MM-49464

File: data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md (hunk 1 above)
Comment (yasserfaraazkhan) [reply data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:49]: @sure Maria
I'll update about Yearly saving not being displayed any more 😄 

the screenshot is a bit tricky... once I merge this, I'll directly update this in Zephyr.
Currently, we don't have a way to update screenshots seamlessly from this repo.

File: data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md [hunk 2]
```diff
@@ -42,19 +42,24 @@ steps_hashed: f690569e888bbcc689e9149511b902ca20999833a5f8e71942c37778242c459ad4
...
 \
 4\. Total Cost should be displayed\
 \
-5\. 'Upgrade' button should be enabled if the Data is correct entered.\
-\
-**For Monthly Option**\
+5\. 'Upgrade' button should be enabled if the Data is correct entered.
```
Comment (lindalumitchell) [inline data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:53]:
```suggestion
5\. 'Upgrade' button should be enabled if the Data is correctly entered.
```

**Review by lindalumitchell — COMMENTED** [review]
Added one minor comment, and would be happy to re-review after any other changes may be made as well. :)

**Review by lindalumitchell — APPROVED** [review]
Thanks, looks great!

File: data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md (hunk 1 above)
Comment (saturninoabril) [reply data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:49]: re: adding/updating screenshots, we discussed previously in [channel](https://community.mattermost.com/core/pl/3z4h6sq8aif79pyak194zyiayo) and agreed to include the image as asset in this repo to own and version control - https://community.mattermost.com/core/pl/3z4h6sq8aif79pyak194zyiayo. Steps in doing so are:
1. Add the image at `data/asset/` folder.
- Image file should have a format to easily reference. Suggested format would be `<test_key>-<short_description>.png`, ex. `MM-T4870-onboarding-task-list.png`.
3. Then in the markdown file, the syntax would be ``.

See this PR from Steve for example - https://github.com/mattermost/mattermost-test-management/pull/57/files

Comment (yasserfaraazkhan) [reply data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:49]: Sure @saturninoabril .

in zephyr should I remove the links or separately upload the shots in zephyr?

Comment (saturninoabril) [reply data/test-cases/cloud/automated-tests/system-console/subscription-/MM-T5128.md:49]: No need to upload separately in Zephyr. Just need to update the links like what you've done below in test data field.

**Comment by yasserfaraazkhan** [comment]
> @yasserfaraazkhan not sure if you will have this in another PR but we also need to add the new notifications current monthly customers will have https://mattermost.atlassian.net/browse/MM-49464

Thank you Maria
I've added a test looking at draft PR

If there will be changes, I'll update it accordingly.

**Review by marianunez — APPROVED** [review]

**Review by saturninoabril — APPROVED** [review]
@yasserfaraazkhan Thanks and LGTM!

For screenshot, I would suggest to set a max limit of file size as this will significantly increase the size of the repo. Maybe max of 500KB?

**Comment by yasserfaraazkhan** [comment]
> @yasserfaraazkhan Thanks and LGTM!
> 
> For screenshot, I would suggest to set a max limit of file size as this will significantly increase the size of the repo. Maybe max of 500KB?

Sure Saturn
I've decreased the sizes :D

**Pull request merged**